paperqa/llms.py

```diff
@@ -524,8 +524,15 @@ def maybe_set_config_attribute(cls, data: dict[str, Any]) -> dict[str, Any]:
                         ),
                     }
                 ],
-                "router_kwargs": {"num_retries": 3, "retry_after": 5, "timeout": 60},
             } | data.get("config", {})
+
+        if "router_kwargs" not in data.get("config", {}):
+            data["config"]["router_kwargs"] = {
+                "num_retries": 3,
+                "retry_after": 5,
+                "timeout": 60,
+            }
```

Comment:
Can we somehow avoid redefining these numbers twice (other place being `settings.make_default_litellm_router_settings`)? Perhaps we can use a module level variable to share the parameters

Reply:
NEVERMIND! 😆